mmv1/third_party/terraform/tests/resource_storage_bucket_test.go.erb

```diff
@@ -1464,6 +1489,20 @@ resource "google_storage_bucket" "bucket" {
 `, bucketName, enabled)
 }
 
+<% unless version == 'ga' -%>
+func testAccStorageBucket_publicAccessPrevention(bucketName string, prevention string) string {
+	return fmt.Sprintf(`
+resource "google_storage_bucket" "bucket" {
+  provider                  = google-beta
+  name                      = "%s"
+  location                  = "US"
+  public_access_prevention  = "%s"
+  force_destroy             = true
+}
+`, bucketName, prevention)
+}
+<% end -%>
```

Comment:
```suggestion
func testAccStorageBucket_publicAccessPrevention(bucketName string, prevention string) string {
	return fmt.Sprintf(`
resource "google_storage_bucket" "bucket" {
  provider                  = google-beta
  name                      = "%s"
  location                  = "US"
  public_access_prevention  = "%s"
  force_destroy             = true
}
`, bucketName, prevention)
}
```

Reply:
Not quite clear on these suggestions... you're implying the tests shouldn't be gated either? Surely we only want to test this behavior for the beta provider, which is the only place it's being added?

Reply:
As with the previous change, if I'm understanding this correctly we don't need any conditionals in our test file as the choice of provider for the resource determines what gets run. Resolving this based on that assumption.